tests/unittests/CoreGraphics.drawing/ImageComparison.h

```diff
@@ -39,7 +41,107 @@ class ImageComparator {
     virtual ImageDelta CompareImages(CGImageRef left, CGImageRef right) = 0;
 };
 
+struct bgraPixel {
+    uint8_t b, g, r, a;
```

Comment:
(I'd recommend that approach in the interim.)

---
In reply to: [95861143](https://github.com/Microsoft/WinObjC/pull/1692#discussion_r95861143) [](ancestors = 95861143)